src/Arc/Clay/Inputs/Checkbox.hs

```diff
@@ -25,7 +25,7 @@ checkboxStyle = do
         cursor pointer
   where
     borderUnchecked = rgba 223 225 230 1
```

Comment:
We should probably have semantics for this as well. Maybe `instance ColourToken Checkbox`, where backgroundColour is the unchecked state, and foregroundColour is checked? Or maybe we should further introduce `active` and `inactive` colours to the typeclass.
What do you think?